sdks/python/scripts/generate_pydoc.sh

```diff
@@ -235,17 +235,6 @@ nitpick_ignore = []
 nitpick_ignore += [('py:class', iden) for iden in ignore_identifiers]
 nitpick_ignore += [('py:obj', iden) for iden in ignore_identifiers]
 nitpick_ignore += [('py:exc', iden) for iden in ignore_references]
-
-# Monkey patch functools.wraps to retain original function argument signature
-# for documentation.
-# https://github.com/sphinx-doc/sphinx/issues/1711
```

Comment:
This Sphinx [issue](https://github.com/sphinx-doc/sphinx/issues/1711) has been fixed, so removing this code